tests/qgis-projects/tests/feature_toolbar.qgs.cfg

```diff
@@ -128,6 +128,15 @@
             "hideLayer": "False",
             "custom_config": "False",
             "order": 0
+        },
+        "children_layer": {
```

Comment:
Is-it a known key `children_layer` ?
I see nothing in the source code of the plugin https://github.com/3liz/lizmap-plugin/search?q=children_layer

Reply:
It is the layer name :)